src/services/store.ts

```diff
@@ -7,6 +7,10 @@ export type Actions = {
 
   adequacies: Adequacies
 
+  adequaciesComputed: boolean
```

Comment:
This seems to be duplicating state. We already know if adequacies are computed if `adequacyComputationInProgress` is `false`.